docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        deprecatedrpc=signrawtransaction
+        addresstype=p2sh-segwit
+        rpcuser=ceiwHEbqWI83
+        rpcpassword=DwubwWsoo3
+  eclair_bitcoin:
+    restart: unless-stopped
+    stop_signal: SIGINT
+    image: acinq/eclair:v0.3
+    environment:
+      PUBLIC_HOST: ${BTCPAY_HOST}
+      JAVA_OPTS: |
+        -Xmx256m
+        -Declair.printToConsole
+        -Declair.headless
```

Comment:
Hey this is not necessary when running eclair-node, which is what acinq/eclair:v0.3 does.

Reply:
I think it's ok to be a little verbose here for a quick glance at settings :)

Reply:
Oh i referred to `-Declair.headless`, it has no effect when passed to eclair-node jar. That option is meant to run eclair-node-gui in headless mode but in the docker we only build eclair-node.